src/app/components/OptimoPromos/Title/index.styles.jsx

```diff
@@ -0,0 +1,14 @@
+import { GEL_SPACING } from '#app/legacy/gel-foundations/src/spacings';
+import { C_GREY_10 } from '#app/legacy/psammead-styles/src/colours';
+import { getSerifMedium } from '#app/legacy/psammead-styles/src/font-styles';
+import styled from '@emotion/styled';
+
+export const TitleNoPadding = styled.div`
```

Comment:
These component describe the appearance, which the standards recommends not to do.

Reply:
Renamed:
TitleNoPadding --> TitleOnly
As this style gets applied when the Title does not have content below it

Reply:
TitleWithPadding --> TitleWithContent
As this style gets applied when there is content below the title